api/src/dto/ContentDto.ts

```diff
@@ -51,10 +53,11 @@ export class ContentDto extends _contentBaseDto {
     video?: Uuid;
 
     @IsDate()
-    @IsNotEmpty()
+    @IsOptional()
     publishDate: number;
 
     @IsDate()
+    // TODO required if status set to published
```

Comment:
Do not think we need to have the expiry date set, unless you want the document to expire?

Notes for CMS functionality:
Would be nice if it is possible to both be able to set a date or number of days.

Reply:
> Do not think we need to have the expiry date set, unless you want the document to expire?

Right, this comment belongs to publishedDate instead.